lib/node_modules/@stdlib/stats/base/dists/weibull/logpdf/benchmark/benchmark.js

```diff
@@ -32,17 +33,25 @@ var logpdf = require( './../lib' );
 
 bench( pkg, function benchmark( b ) {
 	var lambda;
+	var len;
 	var k;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	x = new Float64Array( len );
+	lambda = new Float64Array( len );
+	k = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( EPS, 100.0 );
+		lambda[ i ] = uniform( EPS, 100.0 );
+		k[ i ] = uniform( EPS, 100.0 );
+	}
 	b.tic();
+
 	for ( i = 0; i < b.iterations; i++ ) {
```

Comment:
```suggestion
	b.tic();
	for ( i = 0; i < b.iterations; i++ ) {
```

This change is required throughout the PR.